authors/apps/articles/serializers.py

```diff
@@ -0,0 +1,65 @@
+from rest_framework import serializers
+
+from authors.apps.authentication.models import User
+from authors.apps.authentication.serializers import UserSerializer
+from django.core.exceptions import ObjectDoesNotExist
+
+from .models import Article
+
+
+class CreateArticleSerializer(serializers.ModelSerializer):
+    """Serializes create article requests and creates an article"""
+
+    title = serializers.CharField(
+        max_length=255,
+        min_length=2
+    )
+
+    description = serializers.CharField()
+
+    body = serializers.CharField()
+
+    author_id = serializers.IntegerField()
+
+    class Meta:
+        model = Article
+
+        fields = ['title', 'description', 'body', 'author_id']
+
+    def validate(self, data):
+        title = data.get('title', None)
+        description = data.get('description', None)
+        body = data.get('body', None)
+        author_id = data.get('author_id', None)
+
+        if title is None:
```

Comment:
The number of if statements in this section are two too many.
Please reimplement this in such a way that you have one `serializers.ValidationError` and the last word in `An article must have a ___` is picked up dynamically.

Reply:
Thank you for the feedback. While refactoring, I realized that these validation checks are only enforced if the attributes of the serializer fields are set to `required=False`. I also realized that these checks are redundant since the serializer already does these validations by default. Therefore, in an attempt to remain DRY, I discarded these checks and only use the validate method for adding an author to an article.